### ISSUE TITLE
Navigation Menubar Example accessibility issue

### ISSUE THREAD

**Issue by mintzLB** [w3c/aria-practices#2248]
Referring to the [Navigation Menubar example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/menubar/menubar-navigation.html)

Navigation menubar example does not announce submenus can be expanded, so the submenus are not discoverable with a screen reader. 

Tested with JAWS 2022 and edge Version 98.0.1108.56 and chrome Version 98.0.4758.102 

These are examples of what I am referring to as submenus:
![submenu campus tours expanded](https://user-images.githubusercontent.com/100642822/156076999-abba2675-3882-4df9-87ee-0c00c6e020c8.png)

**Comment by ZoeBijl** [comment]
Confirmed with JAWS 2021 and Chrome 98. Native Windows submenus are announced by JAWS. Is this a mapping issue?

**Comment by JAWS-test** [comment]
https://github.com/FreedomScientific/VFO-standards-support/issues/415

**Comment by a11ydoer** [comment]
this is screen reader issue.
